samples/Azure.Management.Storage/Generated/Extensions/TenantExtensions.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.Management.Storage
+{
+    /// <summary> A class to add extension methods to Tenant. </summary>
+    public static partial class TenantExtensions
+    {
+        #region BlobService
+        /// <summary> Gets an object representing a BlobService along with the instance operations that can be performed on it but with no data. </summary>
+        /// <param name="tenant"> The <see cref="Tenant" /> instance the method will execute against. </param>
+        /// <param name="id"> The resource ID of the resource to get. </param>
+        /// <returns> Returns a <see cref="BlobService" /> object. </returns>
+        public static BlobService GetBlobService(this Tenant tenant, ResourceIdentifier id)
```

Comment:
We should only add this duplicate if the parent is actually tenant

Reply:
Do we need this or just `GetBlobServices()` to return the container?

Reply:
we only delegate when the parent is the tenant, otherwise it just gets placed on the armclient only.

Reply:
Updated